x-pack/plugins/code/public/style/_layout.scss

```diff
@@ -101,6 +109,8 @@
   flex-grow: 1;
   flex-shrink: 1;
   overflow: auto;
+  max-width: 256px;
+  min-width: 256px;
```

Comment:
By setting a min and max width of the same value here, you're essentially setting it to that width always. Therefore you can just set it to `width: 256px`. But if you can, try to use a multiple the `$euiSize` like `$euiSize * $euiSize` = 256.

Reply:
done